addon/addon/components/animation-context/index.ts

```diff
@@ -68,8 +68,8 @@ export default class AnimationContextComponent extends Component<AnimationContex
     this.currentBounds = getDocumentPosition(element);
   }
 
-  shouldAnimate(changeset: Changeset): boolean {
-    return !!(changeset && this.args.use && this.isInitialRenderCompleted);
+  shouldAnimate(): boolean {
```

Comment:
Do we still need this check? Would this be better as an assertion for `isStable` in the `maybeTransitionTask`, and/or a warning about having sprites in the changeset but no `use` argument?

Reply:
Yeah we could refactor further into that, though I don't mind the getter in it's updated form. I removed the changeset thing as we already check that before even creating one for the context.

For the last point: we don't really care if people choose not to animate sprites in the changeset I think.

Reply:
Ya we don't, though given that lowest stable context drives, using an animation context without a `@use` argument seems like it can lead to a "bug" where you don't know why an animation isn't working. I guess warnings like this can wait till the high level API and how we expect it to be used are clearer though?